### ISSUE TITLE
Kanban Feature Request: Have a Default Project/Kanban Board be Shown

### ISSUE THREAD

**Issue by zaha** [go-gitea/gitea#14871]
In most cases I only need a single project/kanban board per repository. I would like to set a kanban board as the default which

- will then be immediately shown when I click on the "Projects" tab
- will be automatically set as the project on any new issues

Right now I have to click "Projects", look for the single board and then click it again, which adds up a lot of unnecessary clicks over time.
Same when creating issues, requiring two additional clicks to add the issue to the board. I can also see this made more comfortable by having this be done by #14824 (new issue button on Kanban Board) alternatively.

**Comment by delvh** [comment]
Well, I can understand your request, but there are two sides to that feature:
1. Those that use a single project for all issues or multiple projects that enable an implicit form of hierarchy - Those are the ones this feature is useful for.
2. Those that use multiple disjoint projects where no default can be set - those are the ones who don't want this feature.

Maybe a compromise could be found, where a default **can** be present but **must not** be. If no default project is present, the current approach should be taken.
What about that?

**Comment by lunny** [comment]
We can add options on repository settings.

**Comment by delvh** [comment]
Yes, exactly. But a `nil` default value should be a valid case.

**Comment by zaha** [comment]
Yes, you are completely right. I wanted to suggest this as an option for people that only needed a single board. At least in my head, seems like I completely forgot to write that down in the description, my bad. 😅 

An option on the repository settings would be perfect! 👍
